cdap-docs/developers-manual/source/testing/testing.rst

```diff
@@ -74,6 +77,23 @@ then we’ll start the flow and the service::
       ServiceManager serviceManager = appManager.startService("RetrieveCounts");
       serviceManager.waitForStatus(true);
       
+This flow counts the words in the events that it receives on the "wordStream". Before
+sending data to the stream, let us validate that we start with a clean state::
+
+      // validate that the wordCount table is empty, and that it has no entry for "world"
+      DataSetManager<KeyValueTable> datasetManager = getDataset(config.getWordCountTable());
+      KeyValueTable wordCounts = datasetManager.get();
+      Assert.assertNull(wordCounts.read("world"));
+
+Note that the dataset manager implicitly starts a transaction for the dataset when it is
+initialized. Under this transaction, read operations can only see changes that were
+committed before the transaction was started; and any writes performed within this
+transaction only become visible after the transaction is committed. This can be done
```

Comment:
I think it should be:
"only become visible **to other transactions** after **this** transaction is committed."